src/_data/products.json

```diff
@@ -25,14 +25,36 @@
   },
   {
     "logo": "products/oab.svg",
-    "name": "OAButton",
+    "name": "OAbutton",
     "url": "https://oabutton.org",
     "label": "Open Access delivery",
     "tagline": "Get around paywalls in a click",
     "cta": "Find an article",
-    "description": "OAButton provides free access to papers legally by scouring the web and emailing authors. Use it on the website, via plugins for Chrome and Firefox, or embed it into your library system.",
+    "description": "OAbutton provides free access to papers legally by scouring the web and emailing authors. Use it on the website, via plugins for Chrome and Firefox, or embed it into your library system.",
     "embed": "<div id='oabutton'><h2>Avoid paywalls, request research</h2><label for='oabutton_url'>Enter an article’s URL, DOI, Title, or Citation</label><script src='https://openaccessbutton.org/static/jquery-1.10.2.min.js'></script><script src='https://openaccessbutton.org/static/openaccessbutton_widget.js'></script><script>jQuery(document).ready(function(){openaccessbutton_widget({uid:'h9mTMgqeaKDyJW8E2'});});</script><div id='openaccessbutton_widget'></div></div>",
     "colour": "chartreuse",
     "featured": true
+  },
+  {
+    "logo": "",
+    "name": "OAreport",
+    "url": "https://oa.report",
+    "label": "Open Access reporting",
+    "tagline": "Put OA policies into practice in minutes, not months.",
+    "cta": "Sign up for updates",
+    "description": "OAreport translates policy to impact by helping libraries and funder discover papers covered by your policy, analyze if and how papers comply, and unlock individual papers and systemic changes.",
```

Comment:
"your policy" → "their policy"

Reply:
That’s what on the OA.report website -- should we also update it there then?

Reply:
Hmm, for some reason on those sites it feels like it's talking to "you", where as here it feels more like "them" –– so I think the change makes sense here but not there.

However, I generally feel like you're better at these sorts of things than me. So, if you disagree feel free to go with your preference.

Reply:
Well the correct way would be "their policy" of course (as suggested here). What’s on the OAreport website is grammatically incorrect.

Reply:
@JosephMcArthur  "Then let's go with their in both places. Thanks again!"

Reply:
Corrected to "by their policies"